### ISSUE TITLE
OBS mode not working

### ISSUE THREAD

**Issue by blackholegalaxy** [pimarc/pms50-gns530#26]
As IFR pilot, when holding, it's usual to be able to track on your holding fix using OBS mode on a given radial. Doing so allow in HSI to hold easily.

The obs mode is not working at all. No track line is drawn on radial (no radial modal requesting for degrees) and HSI is not linked to the OBS mode (it should leave GPS mode).

**Comment by pimarc** [comment]
Yes, Asobo didn't finish the work here.
While in OBS mode, the OBS button of the HSI should allow changing the radial.
This is not a GPS issue. It's at the aircraft level.
I will try to find find a solution in order to give the user a way to set the radial from the GNS530 but this is not the standard way and anyway, the HSI should be able to do that.

In my GTN750 product (coming soon), I was able to do that directly from the GPS.

Asobo made the necessary change only in the G1000 (it has a dedicated CRS button). But still with bugs (bad radial display in TRK UP mode and double lin in directTo).

They have implemented the functionality only since the last update (sim update 3) and I hope they will correct bugs and finish the OBS implementation soon.

I wrote something on the forums about that:
https://forums.flightsimulator.com/t/gtn750-project/377721/73?u=scorpionfilm422
